src/components/sidecar/get-truncated-title.ts

```diff
@@ -0,0 +1,27 @@
+const MAX_CHARACTERS = 26
+
+function getTruncatedTitle(fullTitle: string): string {
+  let truncatedTitle: string
+
+  if (fullTitle.length < MAX_CHARACTERS) {
```

Comment:
I'm not sure -  apart from counting the spaces, I think we'd need to keep `fullTitle.length < MAX_CHARACTERS` and `characterCount + wordLength < MAX_CHARACTERS` consistent (ie both `<` or both `<=`). This might make `MAX_CHARACTERS` more intuitive though (ie it's an inclusive max), so applying it and will update the other comparison as well 👍